lexers/ansi_c.lua

```diff
@@ -51,6 +51,11 @@ local include = lex:tag(lexer.PREPROCESSOR, '#' * S('\t ')^0 * 'include') *
 local preproc = lex:tag(lexer.PREPROCESSOR, '#' * S('\t ')^0 * lex:word_match(lexer.PREPROCESSOR))
 lex:add_rule('preprocessor', include + preproc)
 
+-- Attributes.
+lex:add_rule('attribute', lex:tag(lexer.ATTRIBUTE, '[[' * 
+  (lex:word_match(lexer.ATTRIBUTE) + (lexer.word * ':' * lexer.word)) * 
```

Comment:
I'm pretty sure you meant to use `'::'` instead of `':'`.

Reply:
so true